snapshot/snapshotter.go

```diff
@@ -70,6 +72,7 @@ type Info struct {
 	Labels  map[string]string `json:",omitempty"` // Labels for snapshot
 	Created time.Time         `json:",omitempty"` // Created time
 	Updated time.Time         `json:",omitempty"` // Last update time
+	Options *types.Any        `json:"-"`          // Snapshot specific options
```

Comment:
I don't think this should be omitted when serialized as JSON, but we may need to think about that further.

Reply:
From the comment, i think its only serialized for ctr output